scripts/build.js

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+
+'use strict';
+
+let colors = require('colors'),
+	exec = require('child_process').exec,
+	pkg = require('../package.json'),
+	contributors = pkg.contributors.map(function(contributor) {
+		return contributor.split(/(?:\s<.*>)?\s\(.*\)/)[0];
+	}).join(', '),
+	preamble = `/*! loadJS: ${pkg.description} (c) 2014-present ${contributors} (Based on http://goo.gl/REQGQ by Paul Irish). Freely distributed under the ${pkg.license} license. */`;
```

Comment:
Using ES6's [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here since Node supports that sort of thing on the command line. :sunglasses: